### PR TITLE
inserted missing implementation for some GameAction subclasses

### PR DESCRIPTION
**Pull request by csunday95**
Creating pull request for fixing issue #13615

So far only has SetStaffNameAction implementation, but am planning to browse through the actions and see if any others need the same additions

### CODE REVIEW

File: src/openrct2/actions/StaffSetNameAction.cpp [hunk 1]
```diff
@@ -26,6 +26,12 @@ StaffSetNameAction::StaffSetNameAction(uint16_t spriteIndex, const std::string&
 {
 }
 
+void StaffSetNameAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit("spriteIndex", _spriteIndex);
```
Comment (IntelOrca) [inline src/openrct2/actions/StaffSetNameAction.cpp:31]: I think just `id` would be better than `spriteIndex`. We no longer refer to entities as sprites.

Comment (duncanspumpkin) [reply src/openrct2/actions/StaffSetNameAction.cpp:31]: In `GuestSetNameAction` we used `peep`

Comment (csunday95) [reply src/openrct2/actions/StaffSetNameAction.cpp:31]: I just selected spriteIndex to line up with the member name, would it be preferred to have a more human readable name like "peep" or "peepId" for the outward facing API?

Comment (duncanspumpkin) [reply src/openrct2/actions/StaffSetNameAction.cpp:31]: I think `peep` would be a best. There are at least 2 other actions that use that.

File: src/openrct2/actions/FootpathPlaceAction.cpp [hunk 2]
```diff
@@ -277,8 +277,9 @@ GameActions::Result::Ptr FootpathPlaceAction::ElementInsertQuery(GameActions::Re
 GameActions::Result::Ptr FootpathPlaceAction::ElementInsertExecute(GameActions::Result::Ptr res) const
 {
     bool entrancePath = false, entranceIsSamePath = false;
+    uint32_t flags = GetFlags();
 
-    if (!(GetFlags() & (GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED | GAME_COMMAND_FLAG_GHOST)))
+    if (!(flags & (GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED | GAME_COMMAND_FLAG_GHOST)))
```
Comment (duncanspumpkin) [inline src/openrct2/actions/FootpathPlaceAction.cpp:282]: don't make unrelated changes in this PR save it for a different PR.

Comment (csunday95) [reply src/openrct2/actions/FootpathPlaceAction.cpp:282]: wait... how did this end up here? dang I think I must have been working on the wrong local branch when I was investigating something. I'll revert this on my next commit to this PR

**Review by IntelOrca — APPROVED** [review]

**Comment by csunday95** [comment]
ah shoot I seem to have messed up this PR while trying to rebase something else; not sure what to do in this situation since I committed on top of the rebase and pushed. Might just be best to close this PR and redo it from scratch? redoing these changes is pretty trivial with PR this as a reference

**Comment by duncanspumpkin** [comment]
Just do a rebase of develop. It will remove all of the merge related commits that you have added and keep only your changes.
